src/client/render/render-core.js

```diff
@@ -148,6 +148,54 @@ let renderCore = function() {
 			}
 		},
 
+		setCameraStyle: function(type) {
+			// If this function is called from an event, use the event origin node to determine what type to become
+			if (this && typeof type !== "string") type = this.dataset.type;
+
+			// Check if we're not already the camera type we try to become
+			if (renderCore.controls && type === renderCore.controls.type) return; // Is already this type.
+
+			// Helper function that copies position and rotation from previously used camera
+			function copyPositionAndRotation(target, source) {
+				target.camera.position.copy(source.camera.position);
+				target.camera.rotation.copy(source.camera.rotation);
+			}
+
+			// Copy over transform data, disable previously used camera / controls, enable new camera / controls
+			switch (type) {
+			case "TrackingCamera":
```

Comment:
I realise this was part of the code before, but shouldn't these parameters be without the first capital letter? E.g. `trackingCamera`.

Reply:
I think we could do even better, and approach it the same way as other libraries using constants. How about define the constants in `cameras.js` such as:
```js
cameras.FREE_CAMERA = 1;
cameras.TRACKING_CAMERA = 2;
```
which can then be checked (say, in `render-core.js`) as
```js
import { cameras, FreeCamera, TrackingCamera } from "./cameras";

...

switch (renderCore.activeCamera.type) {
case cameras.TRACKING_CAMERA:
    ...
    break;
}
```
?